chapter/design-principles.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Design Principles</title>
+    <style>
+        blockquote {
+            color: Orange;
+        }
+
+        .info {
+        float: left;
+        padding-left: 30px;
+        padding-right: 30px;
+        padding-top: 40px;
+        padding-bottom: 30px;
+        width: 130px;
+        }
+
+    </style>
+</head>
+
+<body>
+
+    <h1>2.3</h1>
+    <h1>Design Principles</h1>
```

Comment:
Du solltest nur eine `<h1>` in deinem Dokument haben. Ich würde die Kapitel-Nummer und den Titel in ein Tag packen. Das Umbrechen der Zeile machen wir dann mit CSS